[tests] relative time threshold doesn't work for multiple

nanomosfet opened the pull request:
Added breaking test for multiple relative time thresholds

https://github.com/moment/moment/issues/4909

marwahaha commented:
I'm ok checking in a test that documents the existing behavior, but not one that changes the logic.

nanomosfet commented:
I have removed the change to `humanize.js` that changes the logic. and updated the added tests to document the behavior 👍 @marwahaha 

Unfortunately, I think the pipeline will fail because of the failing Japanese locale tests

marwahaha commented:
Thanks!
